src/OpenRiaServices.EntityFramework/Framework/OpenRiaServices.EntityFramework.csproj

```diff
@@ -43,5 +43,9 @@
       <LastGenOutput>MetadataResource.Designer.cs</LastGenOutput>
       <CustomToolNamespace>System.Data.Mapping</CustomToolNamespace>
     </EmbeddedResource>
+    <EmbeddedResource Update="Resource.resx">
```

Comment:
I think this change should be removed or it will make the resource public

Reply:
Removed